mllib/src/main/scala/org/apache/spark/ml/classification/LogisticRegression.scala

```diff
@@ -1471,6 +1471,17 @@ sealed trait LogisticRegressionSummary extends Serializable {
   /** Returns weighted averaged f1-measure. */
   @Since("2.3.0")
   def weightedFMeasure: Double = multiclassMetrics.weightedFMeasure(1.0)
+
+  /**
+   * Convenient method for casting to binary logistic regression summary,
```

Comment:
Tiny grammar nit:
"""
Convenient method for casting to binary logistic regression summary.
This method will throw an Exception if the summary is not a binary summary.
"""

(This will also make tests rerun.)